frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/initial-state/storage-tab-initial-state.ts

```diff
@@ -94,11 +96,16 @@ const getUrlStorage = (storageClassConfigMap: ConfigMapKind) => {
   };
 };
 
-export const getProvisionSourceStorage = (
-  provisionSource: ProvisionSource,
-  iStorageClassConfigMap: any,
-): VMWizardStorage => {
+export const getProvisionSourceStorage = (state: any, id: string): VMWizardStorage => {
```

Comment:
we already have a similar method which is doing something different (`iGetProvisionSourceStorage`). Can we rename this to `getNewProvisionSourceStorage` ?

Reply:
sure :+1: done